src/js/profile/indicator.js

```diff
@@ -21,74 +21,81 @@ export class Indicator extends Component {
 
         isLast = _isLast;
 
-        if (indicatorData.chartConfiguration.rawHtml) {
-            this.addIndicatorText(wrapper, title, indicatorData);
-        }
-        else {
-            this.addIndicatorChart(wrapper, title, indicatorData, detail);
+        this.indicatorNode = indicatorClone.cloneNode(true);
+        $(indicatorTitleClass, this.indicatorNode).text(this.title);
+        $(chartDescClass, this.indicatorNode).text(this.indicatorData.description);
+        this.extractMetadata(this.indicatorData.metadata, this.indicatorNode);
```

Comment:
@michaelglenister could you please move this to a prepareDomElements method (see the Component Architecture page in gitbook)

Reply:
Will do that